#
#  ### Params of the orthopus interface ###
#

#
#  Minimize cartesian velocity coefs : dx (diag matrix 7x7 in case of quaternion and 6x6 in case of euler angles)
#        
alpha_weight:     [4., 4., 4., 1., 1., 1., 1.]

#
#  Minimize joints velocities coefs : dq
#        
beta_weight:      [0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01]

#
#  Saved joint positions 
#        
# home_position:    [0.75, 0.1, 0.2, 0.3, 0.4, 0.5]
home_position:    [0., 0., 0., 0., 0., 0.]
rest_position:    [0.0, 0.64, -1.397, 0.0, 0.0, 0.0]


#home_position:    [-1.5708, 0.0, 0.0, -1.5708, 1.5708, 1.5708]
# home_position:    [-1.5708, 0.45, -0.45, -1.5708, -1.5708, 1.5708]

#
#  Saved cartesian positions 
#  
drink_pose:
  position:
    x: -0.0237
    y: -0.137
    z: 0.402
  orientation:
    w : 0.0
    x : 0.0
    y : 0.0
    z : -1.0

stand_pose:
  position:
    x: 0.0237
    y: -0.126
    z: 0.402
  orientation:
    w : 1.0
    x : 0.0
    y : 0.0
    z : 0.0

reference_frame             : "ground_link"
move_group_commander_name   : "arm"
allow_replanning            : true
goal_joint_tolerance        : 0.1 # rad
goal_position_tolerance     : 0.001 # m
goal_orientation_tolerance  : 0.01 # unit quaternion

# Trajectory controller gain (PI)
traj_ctrl_position_p_gain      : 10.0
traj_ctrl_position_i_gain      : 0.5
traj_ctrl_orientation_p_gain      : 4.0
traj_ctrl_orientation_i_gain      : 0.2

joint_max_vel               : 0.6   # rad/s
space_position_max_vel      : 0.2   # m/s
space_orientation_max_vel   : 0.8   # rad/s

debug                       : true
sampling_frequency          : 10    # Hz

